src/appmixer/snowflake/db/UpdatedRow/UpdatedRow.js

```
@@ -23,16 +23,16 @@ module.exports = {
-            const {
-                // eslint-disable-next-line no-unused-vars
-                METADATA$ACTION: oldAction, METADATA$ISUPDATE: oldUpdate, METADATA$ROW_ID: oldId,
-                ...oldRow
-            } = data[i + mid];
-            const {
-                // eslint-disable-next-line no-unused-vars
-                METADATA$ACTION: updatedAction, METADATA$ISUPDATE: updatedUpdate, METADATA$ROW_ID: updatedId,
-                ...updatedRow
-            } = data[i];
+            const oldRow = { ...data[i + mid] };
+            delete oldRow.METADATA$ACTION;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)